server/controllers/recipesController.js

```diff
@@ -4,33 +4,54 @@ const jwt = require("jsonwebtoken");
 /* Start n8n workflow using ingredients list provided*/
 // POST /api/recipes/suggest
 const suggestRecipes = async (req, res) => {
-  const { recipe } = req.body; // Receive ingredients list from the front-end
-
-  //to be placed inside n8n later
-  const inputprompt = (recipe) => {
-    return `Take the input of ingredients (${recipe}) and clean/organize them into a table. If there is no weight or amount listed, then do not include it in the table. Make it short and don't add explanations. If the input is not in English then translate it to English. Do not keep the other language. Using the ingredients in the table, come up with recipes that could be created using these ingredients. Don't include recipes that require any missing ingredients. Each recipe should contain a name, serving amount, kcal, protein, carbs, fat, cooking time, cuisine type (i.e. French, Japanese, Mexican, etc), and number of ingredients from the table that will be required to cook the recipe. Add a step-by-step cooking guide for the recipe. Avoid generic recipes like simple white rice or toast with jam that only require one or two ingredients. Return each recipe as its own JSON object following the format {'Recipe Name': name of recipe, 'ingredients': necessary ingredients with specific amount, 'cooking_guide': specific step-by-step guide for how to make the recipe, 'nutrient_info': calories, protein, carbs, fat of the recipe, 'cooking time': cooking time in minutes, 'cuisine': type of cuisine (i.e. French, Italian, Korean, Russian, etc), 'serving_amount': how many people the recipe can serve}. Make sure that every primitive value in the json object is inside a string, so that arrays are not inside strings.`;
-  };
+    const { ingredients } = req.body; // Receive ingredients list from the front-end
 
   try {
     // URL of N8n webhook per https://docs.n8n.io/integrations/builtin/core-nodes/n8n-nodes-base.webhook/?utm_source=n8n_app&utm_medium=node_settings_modal-credential_link&utm_campaign=n8n-nodes-base.webhook
     //const webhookUrl = `${process.env.N8N_WEBHOOK_URL}/recommend-recipes`;
-    const webhookUrl = process.env.N8N_WEBHOOK_URL_RECIP_TEST;
+    const webhookUrl = process.env.N8N_WEBHOOK_URL_RECIPE;
```

Comment:
Not sure why the new env variable which also is not found in https://github.com/nicmart-dev/feedmenow/blob/develop/server/example.env
I recommend to revert this change and restore line 11

Reply:
Sorry, I created a few more server env variables to send requests to the testing route on n8n. Line 11 can be revoked to its previous state.

Reply:
Was that fixed?